packages/dendron-next-server/components/graph.tsx

```diff
@@ -17,10 +17,13 @@ import AntThemes from "../styles/theme-antd";
 import GraphFilterView from "./graph-filter-view";
 import {
   GraphConfig,
+  GraphConfigItem,
   GraphEdges,
   GraphElements,
   GraphNodes,
 } from "../lib/graph";
+import { VaultUtils } from "../../common-all/lib";
```

Comment:
watch out for relative imports into workspace packages. see [here](https://github.com/dendronhq/dendron-site/blob/dev/vault/dendron.dev.troubleshooting.md#L12:L12)